packages/terra-site/src/examples/form-select/examples/SelectField.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SelectField from 'terra-form-select/lib/SelectField';
+
+const SelectFieldExample = () => (
+  <SelectField label="T-shirt size" placeholder="Select a size" selectId="select-field" required style={{ width: '300px' }}>
```

Comment:
Need to update this to use `maxWidth`. @mmalaker noted it gets cut off on mobile devices.